src/server/api/routes/create-event.router.js

```diff
@@ -0,0 +1,57 @@
+const express = require('express');
+
+const router = express.Router({ mergeParams: true });
+
+// controllers
+const createEventController = require('../controllers/create_event.controller');
+
+/**
+ * @swagger
+ * /createEvent:
+ *  post:
+ *    summary: Create an event
+ *    description:
+ *      Will create an event.
+ *    produces: application/json
+ *    parameters:
+ *      - in: body
+ *        name: event
+ *        description: The event to create.
+ *        schema:
+ *          type: object
+ *          required:
+ *            - event_type
+ *            - event_date
+ *            - venue
+ *            - description
+ *            - max_participants
+ *          properties:
+ *            event_type:
+ *              type: integer
+ *            event_date:
+ *              type: datetime
+ *            venue:
+ *              type: string
+ *            description:
+ *              type: string
+ *            max_participants:
+ *              type: integer
+ *    responses:
+ *      201:
+ *        description: createEvent created
+ *      5XX:
+ *        description: Unexpected error.
+ */
+router.post('/', (req, res) => {
+  console.log('event posted');
```

Comment:
Remove console.log statements.

Reply:
> Remove console.log statements.

code embedded in events.router.js, sorted